site/kube-addon.md

```diff
@@ -205,21 +207,16 @@ UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 
 ### <a name="configuration-options"></a> Changing Configuration Options
 
-The default configuration settings can be changed by saving and editing the
-addon YAML before running `kubectl apply`. Additional arguments may be
-supplied to the Weave router process by adding them to the `command:`
-array in the YAML file.
+#### Using `cloud.weave.works`
 
-Some parameters are changed by environment variables; these can be
-inserted into the YAML file like this:
+If you are getting your YAML file from `cloud.weave.works` as described above, then you can customise it by passing some of Weave Net's options, arguments and environment variables as query parameters:
 
```

Comment:
If the YAML file is from '`cloud.weave.works` as described above, then you can....